### DISCUSSION TITLE
Is there any possible to add search function for source code

### DISCUSSION THREAD

**Discussion by sola1121** [community/community#37615] [Mobile]
I ususally use github mobile to review my notes, but no search function to locate where I want to look.

As a mobile app, because of the screen limited, write code  is hard, the focus function is browse. So I think a search file content funtion may be a good idea.

**Comment by jfullstackdev** [comment]
as for now, if you never tried using the browser of your mobile and log in to GitHub to have that kind of Desktop view

**Reply by sola1121** [reply]
browser will ask me log in again. I think this app can not only for login authentication.

**Reply by jfullstackdev** [reply]
yup, that's not connected to the mobile app, it's a separate log in, but you will be experiencing what you are seeing on the web

**Comment by stevepopovich** [comment]
Hi @sola1121, yes like @jdevstatic you can use the mobile website to get this feature. We will look into adding this on the mobile app. Thanks!

**Reply by the-programmer** [reply]
I am not 100% shure if this is the original question, but it seems related.
What I am missing is, once you open a file (1027 lines), finding the correct line.
I know the text to search for but I can't search for that text in the mobile file viewer.

If I switch to the (mobile) browser finding the correct line is easy since I can use it's search function.
